commands/root.go

```diff
@@ -249,6 +250,9 @@ func addCommands() {
 			logging_service.LoggingServiceCmd(), constants.DefaultLoggingServiceApiURL,
 		),
 	)
+
+	// CDN
+	rootCmd.AddCommand(cdn.Command())
```

Comment:
Default API Url is `https://cdn.de-fra.ionos.com/distributions` I think we need to add this with a changed default API URL:

 `	rootCmd.AddCommand(funcChangeDefaultApiUrl(cdn.Command(), constants.DefaultCDNApiURL))`